frontend/css/add-listen-modal.less

```diff
@@ -0,0 +1,184 @@
+    border: 1px solid rgba(0, 0, 0, 0.15);
+    background-color: @white-background;
+    max-height: 102px;
+    /*overflow: hidden;*/
+    overflow-y: scroll;
+
+    // Links within the dropdown menu
+    > a,
+    > button {
+      display: block; // necessary for <a> tags
+      width: 100%;
+      background: none;
+      color: inherit;
+      border: none;
+      padding: 5px 20px;
+      font: inherit;
+      cursor: pointer;
+      outline: inherit;
+      text-align: left;
+
+      &:hover {
+        color: #fff;
+        background-color: #353070;
+      }
+    }
+  }
+  .addtrackpill {
+    display: inline-block;
+    background-color: #fff;
+    border-radius: 7px;
+    margin-top: 10px;
+    width: 100%;
+    border: 2px solid;
+    outline: none;
+    > div {
+      display: flex;
+      margin: 5px;
+      justify-content: space-between;
+      // We want pills to be always 32px height or more
+      // Texts are 10px, hence minimum height of 22px here
+      min-height: 22px;
+      span {
+        color: #000000;
+        margin-top: auto;
+        margin-bottom: auto;
+        margin-left: 6px;
+        margin-right: 6px;
+        font-style: normal;
+      }
+      button {
+        background-color: #fff;
+        color: #000000;
+        border: none;
+        border-radius: 100px;
+      }
+    }
+  }
+  .track-info {
+    > div {
+      width: 100%;
+    }
+  }
+  .cross-details {
+    > button {
+      border: 0px;
+      background-color: #353070;
+      color: white;
+      margin: 5px;
+    }
+  }
+  .new-details {
+    margin: 20px 0px;
+    margin-right: 10px;
+    background-color: #353070;
+    width: 100%;
+    border-bottom-right-radius: 12px;
+    border-top-right-radius: 12px;
+    display: flex;
+  }
+  .recording-heading {
+    margin: 0px 15px;
+    > h5 {
+      color: white;
+      font-weight: 700;
+    }
+  }
+  .single-entity {
+    display: flex;
+    margin: auto 15px;
+    > h6 {
+      color: white;
+    }
+  }
+  .entity-heading {
+    font-weight: 700;
+    margin: 0px;
+  }
+  .entity-details {
+    font-size: 12px;
+    margin: 0px 18px;
+  }
+  .cover-art-img {
+    margin: 20px 10px;
+    margin-right: 0px;
+    > img {
+      width: 98px;
+      height: 98px;
+      border-bottom-left-radius: 12px;
+      border-top-left-radius: 12px;
+    }
+  }
+
+  .timestamp {
+    margin-left: 18px;
+    margin-top: -10px;
+    > h5 {
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+  .timestamp-entities {
+    display: flex;
+    > button {
+      border-radius: 15px;
+      height: 30px;
+      margin-right: 15px;
+      outline: none;
+      padding: 0px 30px;
+    }
+  }
+  .timestamp-active {
+    background-color: #353070;
+    color: #fff;
+  }
+  .timestamp-unactive {
+    background-color: #fff;
+    border: 2px solid rgb(141, 141, 141);
+    color: #000000;
+  }
+  .btn-success {
+    background: #353070;
+  }
+  .btn-success[disabled] {
+    background: #353070;
+  }
```

Comment:
This is a no-no. We have specific classes from Bootstrap CSS that we can use: https://getbootstrap.com/docs/3.3/css/#buttons-options

But I think keeping the green button might be the best solution as that seems to be what we do in all other modals.